### ISSUE TITLE
unable to download videos from api

### ISSUE THREAD

**Issue by monty-dev** [drawrowfly/tiktok-scraper#507]
**Describe the bug**

## If fields below will be empty then issue will be Ignored and Closed

**To Reproduce**
- Used OS
Oracle Linux 8.3
- NodeJs Version (node -v)
-v15.12.0
- Scraper Version (tiktok-scraper --version)
1.4.20
- CLI or Module
-CLI
- Used method
video "https://www.tiktok.com/@bellapoarch/video/6943622551369305349?sender_device=pc&sender_web_id=6945163207821854213&is_from_webapp=v3&is_copy_url=0" -d

The following output:

Error
    at TikTokScraper.getVideoMetadataFromHtml (/home/x/tiktok-scraper/build/core/TikTok.js:775:15)
    at async TikTokScraper.getVideoMeta (/home/x/tiktok-scraper/build/core/TikTok.js:804:25)
    at async Object.exports.video (/home/x/tiktok-scraper/build/entry.js:186:20)
    at async startScraper (/home/x/tiktok-scraper/bin/cli.js:44:29)

**Screenshots**

**Comment by lau1944** [comment]
To, downgrade module version to 1.4.16 works

**Comment by drawrowfly** [comment]
Fixed in @1.4.22

**Comment by youngvo** [comment]
@monty-dev are you guys using session to download? if yes, let include the session like the follow:
```
let options = {
		sessionList: ['sid_tt=upwdsdldldhaldadsadjk; sessionid=upwdsdldldhaldadsadjk; sessionid_ss=upwdsdldldhaldadsadjk;'],
	};
``` 
You have to include both sid_tt, sessionid and sessionid_ss
